files/common/lib/systemd/system/rtslib-fb-targetctl.service.d/override.conf

```diff
@@ -1,3 +1,6 @@
+[Unit]
+After=zfs-mount.service
```

Comment:
Why are you adding this dependency?

Reply:
Ah, I will add a comment here. This is a temporary substitute for the work that should be done in https://jira.delphix.com/browse/DLPX-64724.

Reply:
I think it might be better to wait on `zfs.target` instead? Since, this makes me think we specifically care about the mounts, which isn't true.. idk if using `zfs.target` will impact anything practically, but it seems more appropriate conceptually (at least until we fix DLPX-64724).

Reply:
I was about to add this comment:
```
#
# This service depends on zvol dev links to be present, which should
# be implemented in DLPX-64724. In the meanwhile, we have this service
# depend on zfs-mount.service, which doesn't guarantee that the zvol
# links will be there, but gives a time buffer after pools have been
# loaded to create the zvol links. This new dependency is especially
# important since local-fs.target, which this service depends on, no
# longer depends on zfs-mount.service.
#
```

In any case I'm working on fixing DLPX-64724 right now, so maybe I'll just update this if I get it fixed first.

Reply:
If we don't fix DLPX-64724 first, I still think using `zfs.target` would be better than `zfs-mount.service`.. `zfs.target` should activate after `zfs-mount.service`, so practically I think it's the same.. But I guess I'm fine with this way too.

Reply:
Following my tests in https://github.com/delphix/zfs/pull/56, I've changed this to `zfs.target` as it is indeed better.